README.md

```diff
@@ -61,16 +61,26 @@ There five categories of metrics that Moneo monitors:
 
 Minimum Requirements
 -----
-
 - python >=3.7 installed
-- docker installed
-- ansible installed (python module)
+
 - OS Support:
-    - Ubuntu 18.04, 20.04
+    - Ubuntu 18.04, 20.04, 22.04
     - AlmaLinux 8.6
+### Manager node requirements
+- docker
+- parallel-ssh
```

Comment:
do we need to fix a parallel-ssh version as well?

Reply:
yea let me address

Reply:
addressed